apps/condo/domains/miniapp/components/AppFrameWrapper.tsx

```diff
@@ -21,9 +14,40 @@ export const AppFrameWrapper: React.FC = ({ children }) => {
         return () => observer.unobserve(document.body)
     }, [])
 
+    useEffect(() => {
+        const tagId = 'fix-body-height-style-tag'
+        const counterAttrName = 'same-style-requests-counter'
+
+        let styleEl = document.querySelector(`#${tagId}`) as HTMLStyleElement | null
+
+        if (styleEl) {
+            const count = parseInt(styleEl.getAttribute(counterAttrName) || '0', 10)
+            styleEl.setAttribute(counterAttrName, String(count + 1))
+        } else {
+            styleEl = document.createElement('style')
+            styleEl.id = tagId
+            styleEl.textContent = 'body{height:auto;}'
+            styleEl.setAttribute(counterAttrName, '1')
+            document.head.appendChild(styleEl)
+        }
+
+        return () => {
+            const styleEl = document.querySelector(`#${tagId}`) as HTMLStyleElement | null
+            if (styleEl) {
+                const count = parseInt(styleEl.getAttribute(counterAttrName) || '1', 10)
+                if (count <= 1) {
+                    if (styleEl.parentNode) {
```

Comment:
Why not just head? You know exactly where it placed, don't you?

Reply:
I mean, does it matter what this parent is at this line of code?